tests/flow/async/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -356,13 +357,15 @@ function test3() {
 //
 
 function test4() {
-  async function voidoid5(): void { // error, void != Promise<void>
+  async function voidoid5(): void {
```

Comment:
Previously the comment was at the end of the line and now we put it on the next line. In this case, the previous algorithm worked as intended. But, I'd argue that the new behavior is very reasonable.